registry/registry.go

```diff
@@ -364,7 +364,7 @@ func configureLogging(ctx context.Context, config *configuration.Configuration)
 			Formatter: &logrus.JSONFormatter{TimestampFormat: time.RFC3339Nano},
 		})
 	default:
-		// just let the library use default on empty string.
+		// unsupported formatter
```

Comment:
Looks like the old comment was an incorrect copy/paste. Might be better to remove the comment altogether. (It's clear from the code below that it's invalid.

Given that we already set a default (if it's empty, we set it to `"text"`) this `if` condition overall looks redundant.